docs/reference/configuration.rst

```diff
@@ -309,6 +309,53 @@ multiple paragraphs, if necessary. The long description *must not* be a copy of
 the ``description``, or include the ``description`` as the first line of the
 ``long_description``.
 
+``requirement_installer_args``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+A list of strings of arguments to pass to the requirement installer when building the app.
+
+Strings will be automatically transformed to absolute paths if they appear to be relative paths
+(i.e., starting with ``./`` or ``../``) and resolve to an existing path relative to the app's
+configuration file. This is done to support build targets where the requirement installer
+command does not run with the same working directory as the configuration file.
+
+If you encounter a false-positive and need to prevent this transformation,
+you may do so by using a single string for the argument name and the value.
+Arguments starting with ``-`` will never be transformed, even if they happen to resolve to
+an existing path relative to the configuration file.
+
+The following examples will have the relative path transformed to an absolute one when Briefcase
+runs the requirement installation command if the path ``wheels`` exists relative to the configuration file:
+
+.. code-block:: TOML
+
+    requirement_installer_args = ["--find-links", "./wheels"]
+
+    requirement_installer_args = ["-f", "../wheels"]
+
+On the other hand, the next two examples avoid it because the string starts with ``-``, does not start with
+a relative path indication (``./`` or ``../``), or do not resolve to an existing path:
+
+.. code-block:: TOML
+
+    requirement_installer_args = ["-f./wheels"]
+
+    requirement_installer_args = ["--find-links=./wheels"]
+
+    requirement_installer_args = ["-f", "wheels"]
+
+    requirement_installer_args = ["-f", "./this/path/does/not/exist"]
+
+.. admonition:: Supported arguments
+
+    The arguments supported in ``requirement_installer_args`` depend on the requirement installer backend.
+
+    The only currently supported requirement installer is ``pip``. As such, the list should only contain valid
```

Comment:
We prefer wrapping both code and documentation at 88 columns. This also makes PR diffs easier to read on GitHub on laptop-size screens with a directory tree sidebar.

Reply:
Thanks for letting me know. flake8 is configured for max line length of 119. I suppose if 88 is the expectation, it's worth changing the linter to catch this so it doesn't need to come up in reviews.

Reply:
Fixed in https://github.com/beeware/briefcase/pull/2059/commits/2807664b3406436f505a181f8367761b9d10543e